### PR TITLE
Sandbox sync tools

### PR DESCRIPTION
**Pull request by pbking**
Now that the GIT and SVN repositories are in sync I'm working on some tooling and workflows to ensure that they stay that way.

#### Changes proposed in this Pull Request:

This change adds a 'sandbox.sh' script that helps to move LOCAL changes to SVN both for DEVELOPMENT or to DEPLOY CODE in preparation for production.

USAGE of the script is outlined in updates to README.md

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
When I run the command for the first time I get this at the end:

```
Writing .sandbox-config
sandbox.sh: line 33: [: ==: unary operator expected
sandbox.sh: line 41: [: ==: unary operator expected
No known command given. [clean, push]
```

When I run clean I get:

```
sh sandbox.sh clean
Pseudo-terminal will not be allocated because stdin is not a terminal.
```

Still, both commands seem to be working ok. The config file is created, the clean command seems to be working fine.

**Comment by MaggieCabrera** [comment]
This is not related to this commit per se, but with the GH sync. When I use the push command this is what gets pushed to SVN:
```
M       README.md
M       apostrophe-2/index.php
M       brompton/package-lock.json
M       brompton/package.json
M       brompton/sass/_extra-child-theme.scss
M       brompton/sass/style-child-theme.scss
M       brompton/style-rtl.css
M       brompton/style.css
M       package-lock.json
M       package.json
?       sandbox.sh
?       seedlet/assets/sass/blocks/blog-posts
?       spearhead/spearhead.zip
```

`apostrophe-2/index.php` is the only file I made a change to test the push. I'm doing this from this branch after rebasing trunk

**Comment by MaggieCabrera** [comment]
After all my testing I did a clean again and I get a long list of files being reverted from our previous sync.

Here is such list: 27281-pb/

**Comment by pbking** [comment]
@MaggieCabrera I believe that the issues you raised are addressed if you have an opportunity to try that again.

**Review by scruffian — APPROVED** [review]
This looks great. 🚢

File: .sandbox-ignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.git
+.github
+.gitignore
+.sandbox-config
+.sandbox-ignore
+sandbox.sh
+vendor
+node_modules
```
Comment (jffng) [inline .sandbox-ignore:8]: Could we add `*.zip` to this ignore?

```suggestion
node_modules
*.zip
```

Also on my first time sync'ing, this repo's README.md is pushed up.

**Review by jffng — APPROVED** [review]
This is so helpful! 

In the future it may be useful to extract this into an npm package or standalone repo so that it can be incorporated in other directories, but in the meantime this LGTM 🚢

**Pull request merged**